types.go

```diff
@@ -1353,6 +1354,22 @@ func (rr *CSYNC) len(off int, compression map[string]struct{}) int {
 	return l
 }
 
+// SVCB RR. See RFC xxxx (https://tools.ietf.org/html/draft-ietf-dnsop-svcb-httpssvc-01)
+// Named ESNI and numbered 0xff9f = 65439 according to draft-ietf-tls-esni-05
+type SVCB struct {
+	Hdr      RR_Header
+	Priority uint16
+	Target   string   `dns:"domain-name"`
+	Value    []string `dns:"txt"` // if priority == 0 this is empty
```

Comment:
this will encode to the wrong wire format

Reply:
What I've just read:

> Values are in a format specific to the SvcParamKey.  Their definition
   should specify both their presentation format and wire encoding
   (e.g., domain names, binary data, or numeric values).

...

>    The "Service Binding (SVCB) Parameter Registry" defines the name
   space for parameters, including string representations and numeric  SvcParamKey values.  This registry is shared with other SVCB-
   compatible RR types, such as HTTPSSVC.

>   ACTION: create and include a reference to this registry.

Which means the wire format of values of key=value pairs depend on keys, and key- value-wireformat-type table was not written which means no one can implement wire format encoding/decoding before the spec is ready.